Old Bailey Online.js

```diff
@@ -259,81 +184,46 @@ var testCases = [
 	},
 	{
 		"type": "web",
-		"url": "https://www.oldbaileyonline.org/browse.jsp?name=OA17110421",
-		"items": [
-			{
-				"itemType": "book",
-				"title": "Ordinary's Account.  21st April 1711",
-				"creators": [],
-				"date": "1711-04-21",
-				"extra": "Reference Number: OA17110421",
-				"libraryCatalog": "Old Bailey Online",
-				"place": "London",
-				"url": "https://www.oldbaileyonline.org/browse.jsp?name=OA17110421",
-				"attachments": [
-					{
-						"title": "OBO Snapshot",
-						"mimeType": "text/html"
-					}
-				],
-				"tags": [],
-				"notes": [],
-				"seeAlso": []
-			}
-		]
-	},
-	{
-		"type": "web",
-		"url": "https://www.oldbaileyonline.org/browse.jsp?name=17100418",
-		"items": [
-			{
-				"itemType": "book",
-				"title": "Old Bailey Proceedings.  18th April 1710",
-				"creators": [],
-				"date": "1710-04-18",
-				"extra": "Reference Number: 17100418",
-				"libraryCatalog": "Old Bailey Online",
-				"place": "London",
-				"url": "https://www.oldbaileyonline.org/browse.jsp?name=17100418",
-				"attachments": [
-					{
-						"title": "OBO Snapshot",
-						"mimeType": "text/html"
-					}
-				],
-				"tags": [],
-				"notes": [],
-				"seeAlso": []
-			}
-		]
-	},
-	{
-		"type": "web",
-		"url": "https://www.oldbaileyonline.org/browse.jsp?id=t16780828-12&div=t16780828-12&terms=hog#highlight",
+		"url": "https://www.oldbaileyonline.org/record/t18000115-12",
 		"items": [
 			{
 				"itemType": "case",
-				"caseName": "[no title]",
+				"caseName": "Trial of Peter Asterbawd, Andrew Forsman: Theft > Burglary, 15th January 1800",
 				"creators": [],
-				"dateDecided": "1678-08-28",
-				"docketNumber": "t16780828-12",
-				"extra": "Reference Number: t16780828-12",
-				"url": "https://www.oldbaileyonline.org/browse.jsp?id=t16780828-12&div=t16780828-12&terms=hog#highlight",
-				"attachments": [
-					{
-						"title": "OBO Snapshot",
-						"mimeType": "text/html"
-					}
-				],
+				"dateDecided": "1800-01-15",
+				"court": "Central Criminal Court, London",
+				"docketNumber": "t18000115-12",
```

Comment:
The old translator always included docketNumber and dateDecided (or at least since before I ever did any work on it). Court is new but seemed worth adding.